data/kr/README.MD

```diff
@@ -0,0 +1,2 @@
+2024/11/21
+Translator : JinGul Lee
```

Comment:
Not sure this file brings any value so far.

Adding change date and change author directly into the project content is a redundant practice since the introduction of git: git is tracking everything for us. With modern tools, you can easily find back accurate data about who and when did a very specific change (search for "Git blame", it's an interesting feature if you work with an IDE like Pycharm or Visual Studio Code).

We are coming back to my "make atomic changes" comment: if each change is made as part of an individual commit, then we get a nice history of changes!

Reply:
What I will consider though is adding you in the CREDITS file for crediting the work you did 😃